src/meta/meta_table.c

```diff
@@ -16,14 +16,18 @@ static bool
 __metadata_turtle(const char *key)
 {
 	switch (key[0]) {
+	case 'C':
+		if (strcmp(key, WT_METADATA_COMPAT) == 0)
+			return (true);
+		break;
 	case 'f':
 		if (strcmp(key, WT_METAFILE_URI) == 0)
 			return (true);
 		break;
 	case 'W':
-		if (strcmp(key, "WiredTiger version") == 0)
+		if (strcmp(key, WT_METADATA_VERSION) == 0)
 			return (true);
-		if (strcmp(key, "WiredTiger version string") == 0)
+		if (strcmp(key, WT_METADATA_VERSION_STR) == 0)
 			return (true);
 		break;
 	}
```

Comment:
Interesting, so there's no error from existing code for unexpected keys...  That makes this change much less scary.

Reply:
I'm not following. The macros (which already existed) should be an exact literal replacement here.

Reply:
Sorry, my point (noticed here but not specifically tied to a line of code) was that there's no code that will result in an error because of an unrecognized key in the turtle file: we just skip over them.